### DISCUSSION TITLE
No available formula with name 'helmwave' while installing vai brew in ubuntu

### DISCUSSION THREAD

**Discussion by tony-sol** [helmwave/helmwave#564] [Q&A] [answered]
I use ubuntu via WSL2 with homebrew inside and macos with homebrew as well.
In macos installation method as
```
brew tap helmwave/tap
brew update
brew install helmwave/tap/helmwave
```
is valid, but not for homebrew on linux:
![image](https://user-images.githubusercontent.com/33671815/226192978-cce9d50e-056c-41d2-9862-fc63d69508fd.png)

According [your tap](https://github.com/helmwave/homebrew-tap/blob/main/formula/helmwave.rb#L30). there is no problem in brew4linux usage, but still - no available formula

**Comment by r3nic1e** [comment]
Sorry for long wait, will check it.
I think I faced the same when I had windows machine but I don't have it now :( Will figure it out

**Reply by r3nic1e** [reply, marked as answer]
Okay, I've found the issue. `goreleaser` produces `formula` directory in our repo but it should be `Formula`.
MacOS FS is case-insensitive by default. But if you use case-sensitive FS it will fail.

I've made a fix - https://github.com/helmwave/helmwave/pull/600
The issue will be resolved as soon as 0.27.0 is released. For a workaround you could just download the formula itself and `brew install ./helmwave.rb`
